packages/eid/app/helpers/getPatientUUID.ts

```diff
@@ -2,17 +2,29 @@ import config from "@amrs-integrations/core";
 let CM = config.ConnectionManager.getInstance();
 
 export default class GetPatient {
-  async getPatientUUIDUsingIdentifier(params: string) {
+  async getPatientUUIDUsingIdentifier(params: string, isCCC: boolean) {
+    let extraParams = "";
+    if (isCCC) {
+      extraParams = `and pi.identifier_type=28`;
+    }
     let amrsCON = await CM.getConnectionAmrs();
-    let sql = `select p.uuid from amrs.patient_identifier pi left join amrs.person p on p.person_id=pi.patient_id where pi.identifier='${params}' and pi.voided=0`;
-    let result: string = await CM.query(sql, amrsCON);
+    let sql = `select p.uuid from amrs_migration.patient_identifier pi left join amrs_migration.person p on p.person_id=pi.patient_id where pi.identifier='${params}' and pi.voided=0 ${extraParams}`;
+    let result: any = await CM.query(sql, amrsCON);
     await CM.releaseConnections(amrsCON);
     return result;
   }
+
   async getPatientOrderNumber(params: string) {
     let amrsCON = await CM.getConnectionAmrs();
-    let sql = `select order_id,encounter_id from amrs.orders where order_number='${params}' and voided=0`;
-    let result: string = await CM.query(sql, amrsCON);
+    let sql = `select order_id,encounter_id from amrs_migration.orders where order_number='${params}' and voided=0`;
```

Comment:
We can now reference amrs so we can close the pr.

Reply:
done